src/components/input-field/input-field.tsx

```diff
@@ -458,11 +456,27 @@ export class InputField {
             return false;
         }
 
+        if (this.type === 'urlAsText' && this.isUrlInvalid(this.value)) {
+            return true;
+        }
+
         const element = this.getInputElement();
 
         return !(element && element.checkValidity());
     }
 
+    private isUrlInvalid(url: string) {
+        if (!url) {
+            return false;
+        }
+
+        if (url.indexOf('.') === -1 && url.indexOf('/') !== 0) {
```

Comment:
Ah, yeah… All these newfangled functions all the kids use nowadays… 😝

Definitely better! I'll update it to match the current millennium 👍

Reply:
⚡

Reply:
I had changed `url.indexOf('.') === -1` to `url.includes('.')` instead of `!url.includes('.')`, which caused the tests to fail. I've now pushed a fix for that 🙂